operator/v1/types_ingress.go

```diff
@@ -145,6 +145,14 @@ type IngressControllerSpec struct {
 	//
 	// +optional
 	TLSSecurityProfile *configv1.TLSSecurityProfile `json:"tlsSecurityProfile,omitempty"`
+
+	// routeAdmission defines a policy for handling new route claims (for example,
+	// to allow or deny claims across namespaces).
+	//
+	// The empty, the default is Strict.
```

Comment:
Actually, does it make sense to state what the default value for `spec.routeAdmission.namespaceOwnership` is in the description of `spec.routeAdmission`?  I think the note could be omitted here entirely.

Reply:
I agree about repeating the defaults, but when this field is nil, "some" default will be applied. I  updated the text to try and describe that without repeating information. PTAL